libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -229,24 +219,69 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
      */
     private _setContainerPosition(): void {
         if (this.contentComponent) {
-            this.contentComponent.getElementRef().nativeElement.style.top = this.header.nativeElement.offsetHeight + 'px';
-            this.contentComponent.getElementRef().nativeElement.style.position = 'relative';
+            const contentComponentElement = this.contentComponent.getElementRef().nativeElement;
+
+            contentComponentElement.style.top = this.header.nativeElement.offsetHeight + 'px';
+            this._addClassNameToCustomElement(contentComponentElement, 'content-sticker');
```

Comment:
There is no need to add `top` anywhere. You can easily do it without applying `top` by applying static/relative positions

Reply:
removed usage of top. Thanks for showing me this approach, Jedrzej!